main/upower/APKBUILD

```diff
@@ -1,17 +1,23 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=upower
-pkgver=0.99.7
+pkgver=0.99.10
+_distver="$(echo ${pkgver} | tr '.' '_')"
```

Comment:
```suggestion
_distver="${pkgver//\./_}"
```

Reply:
Ah, APKBUILD warned me that that's a BASHism, isn't it?

Reply:
You are correct that's not standard, but we use it all over aports anyway.

Reply:
Alrighty then